clearwater-diags-monitor/usr/share/clearwater/bin/clearwater_diags_monitor

```diff
@@ -315,7 +315,17 @@ get_ntp_status()
 # system.
 get_cw_package_checksums()
 {
-  # This isn't supported function under RPM, so only do it for Debian
+  # For Centos the --verify flag for the rpm command returns several
+  # verification diags, including MD5 checksum problems (labelled with a `5`).
+  {
+    if which rpm > /dev/null ; then
+    for pkg in $(clearwater_packages)
```

Comment:
Indent everything in the if block by 2 spaces so that it's easier to see the code flow.